src/components/XdrViewer.js

```diff
@@ -0,0 +1,193 @@
+  'LedgerKeyOffer': 'LedgerKeyOffer',
+  'LedgerKeyTrustLine': 'LedgerKeyTrustLine',
+  'LedgerUpgrade': 'LedgerUpgrade',
+  'LedgerUpgradeType': 'LedgerUpgradeType',
+  'ManageOfferEffect': 'ManageOfferEffect',
+  'ManageOfferOp': 'ManageOfferOp',
+  'ManageOfferResult': 'ManageOfferResult',
+  'ManageOfferResultCode': 'ManageOfferResultCode',
+  'ManageOfferSuccessResult': 'ManageOfferSuccessResult',
+  'ManageOfferSuccessResultOffer': 'ManageOfferSuccessResultOffer',
+  'Memo': 'Memo',
+  'MemoType': 'MemoType',
+  'MessageType': 'MessageType',
+  'NodeId': 'NodeId',
+  'OfferEntry': 'OfferEntry',
+  'OfferEntryExt': 'OfferEntryExt',
+  'OfferEntryFlags': 'OfferEntryFlags',
+  'Operation': 'Operation',
+  'OperationBody': 'OperationBody',
+  'OperationMeta': 'OperationMeta',
+  'OperationResult': 'OperationResult',
+  'OperationResultCode': 'OperationResultCode',
+  'OperationResultTr': 'OperationResultTr',
+  'OperationType': 'OperationType',
+  'PathPaymentOp': 'PathPaymentOp',
+  'PathPaymentResult': 'PathPaymentResult',
+  'PathPaymentResultCode': 'PathPaymentResultCode',
+  'PathPaymentResultSuccess': 'PathPaymentResultSuccess',
+  'PaymentOp': 'PaymentOp',
+  'PaymentResult': 'PaymentResult',
+  'PaymentResultCode': 'PaymentResultCode',
+  'PeerAddress': 'PeerAddress',
+  'Price': 'Price',
+  'PublicKey': 'PublicKey',
+  'ScpBallot': 'ScpBallot',
+  'ScpEnvelope': 'ScpEnvelope',
+  'ScpNomination': 'ScpNomination',
+  'ScpQuorumSet': 'ScpQuorumSet',
+  'ScpStatement': 'ScpStatement',
+  'ScpStatementConfirm': 'ScpStatementConfirm',
+  'ScpStatementExternalize': 'ScpStatementExternalize',
+  'ScpStatementPledges': 'ScpStatementPledges',
+  'ScpStatementPrepare': 'ScpStatementPrepare',
+  'ScpStatementType': 'ScpStatementType',
+  'SequenceNumber': 'SequenceNumber',
+  'SetOptionsOp': 'SetOptionsOp',
+  'SetOptionsResult': 'SetOptionsResult',
+  'SetOptionsResultCode': 'SetOptionsResultCode',
+  'Signer': 'Signer',
+  'SimplePaymentResult': 'SimplePaymentResult',
+  'StellarMessage': 'StellarMessage',
+  'StellarValue': 'StellarValue',
+  'StellarValueExt': 'StellarValueExt',
+  'ThresholdIndices': 'ThresholdIndices',
+  'TimeBounds': 'TimeBounds',
+  'Transaction': 'Transaction',
+  'TransactionExt': 'TransactionExt',
+  'TransactionHistoryEntry': 'TransactionHistoryEntry',
+  'TransactionHistoryEntryExt': 'TransactionHistoryEntryExt',
+  'TransactionHistoryResultEntry': 'TransactionHistoryResultEntry',
+  'TransactionHistoryResultEntryExt': 'TransactionHistoryResultEntryExt',
+  'TransactionResultCode': 'TransactionResultCode',
+  'TransactionResultExt': 'TransactionResultExt',
+  'TransactionResultPair': 'TransactionResultPair',
+  'TransactionResultResult': 'TransactionResultResult',
+  'TransactionResultSet': 'TransactionResultSet',
+  'TransactionSet': 'TransactionSet',
+  'TrustLineEntry': 'TrustLineEntry',
+  'TrustLineEntryExt': 'TrustLineEntryExt',
+  'TrustLineFlags': 'TrustLineFlags',
+  'Uint64': 'Uint64',
+};
```

Comment:
I changed it recently to: https://github.com/stellar/xdr-viewer/blob/a75bf97/app/scripts/app.js#L3 so it updates itself when new XDR structures are added.